client/src/components/PhotoModalDescriptionContainer.jsx

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
+import '../../dist/styles.css';
 
 const PhotoModalDescriptionContainer = (props) => {
   const { photos, currentPhoto } = props;
   const description = photos.length ? photos[currentPhoto].description : "";
   return (
-    <div className="photoModuleDescriptionContainer">
-      <div className="photoModulePhotoNumber">
+    <div className="photoModalDescriptionContainer">
+      <div className="photoModalPhotoNumber">
         {`${currentPhoto + 1} / ${photos.length}`}
```

Comment:
I'd probably make this a variable outside of the render function for cleanliness inside the render